Resources/Prototypes/Loadouts/Generic/items.yml

```
@@ -693,6 +693,19 @@
 
 #Misc Items
+- type: loadout
+  id: LoadoutCivilianDisabler
+  category: Items
+  cost: 5
+  canBeHeirloom: true
+  requirements:
+    - !type:CharacterJobRequirement
+      inverted: true
+      jobs:
+      - Prisoner
```

YAMLlint (1.35.1)

[error] wrong indentation: expected 8 but found 6

(indentation)